adapter/BUILD

```diff
@@ -27,3 +28,17 @@ go_library(
         "//pkg/adapter:go_default_library",
     ],
 )
+
+inventory_library(
```

Comment:
I don't think this mechanism is going to be the final form.

Having a single build rule named `mixer_binary` that takes deps and adapter package arguments and templates is still the goal. But it doesn't exist yet. This is most definitely a placeholder for such a rule (which would likely dispatch to the inventory.bzl script).